src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Match_StronglyTypedEnums_StronglyTypedEnumsDisabled.txt

```diff
@@ -0,0 +1,98 @@
+﻿#SETUP: RegEx
+
```

Comment:
This file is named "Match_StronglyTypedEnums_StronglyTypedEnumsDisabled.txt", but it doesn't need any of the StronglyTypedEnum logic. Can this be merged into the "base" Match.txt?

Reply:
Cleaned up.